src/languages/id-ID.js

```diff
@@ -432,7 +432,7 @@ export default {
     User: "Pengguna",
     Installed: "Terpasang",
     "Not installed": "Tidak terpasang",
-    Running: "Berlari",
+    Running: "Berjalan",
```

Comment:
From what I have looked up, this seems to mean walk over run, with the previous version meaning run?

Reply:
run in term sport, is run, but in term of something start its walk